src/assets/css/main.css

```diff
@@ -204,6 +204,11 @@ blockquote {
   height: 5rem;
 }
 
+.mobile-thumbnail-container .thumbnail {
+  width: 100%;
```

Comment:
Bootstrap has a `w-100` and `w-auto`, could you try those?

Reply:
Not on the `.thumbnail` element itself, which doesn’t know whether it’s in mobile or wide view.